app/controllers/member_authentications_controller.rb

```diff
@@ -26,7 +26,8 @@ def create
 
     if auth.valid?
       flash[:notice] = I18n.t('member_registration.check_email')
-      path = follow_up_member_facing_page_path(page.slug, member_id: auth.member_id)
+      path = PageFollower.new_from_page(page, member_id: auth.member_id).follow_up_path
```

Comment:
In general, just using `new` for creating instances instead of `new_from_page` in this case (unless there's a reason/advantage for having `new_from_page`).